packages/language-core/schemas/vue-tsconfig.deprecated.schema.json

```diff
@@ -51,6 +51,11 @@
 				"experimentalComponentOptionsWrapperEnable": {
 					"deprecated": true
 				},
+				"experimentalAdditionalLanguageModules": {
+					"deprecated": true,
+					"type": "array",
+					"markdownDescription": "https://github.com/vuejs/language-tools/pull/2267"
+				},
```

Comment:
The purpose of marking it as deprecated is to be able to prompt warnings. I would remove the description, but I agree it would be better to deprecate the message.

<img width="573" alt="image" src="https://github.com/vuejs/language-tools/assets/16279759/35f713a7-bfbe-4432-8c12-960714a12638">

Reply:
I've assumed that the schema uses `"additionalProperties": false` so that the removal would raise and error. But I see that it's not the case. I think that adding ti would make sense so that this could be removed and also people see issues if they mistype some option,.

Reply:
I couldn't get `"additionalProperties": false` to work, please feel free to open a PR if you get it to work.

Reply:
`additionalProperties` might not work if there are multiple schemas extending the same object but then my question is: do deprecated properties really need to be separated into its own file? If they wouldn't then it would be easy to use `additionalProperties: false`.

BTW. `deprecated` is not a meant to be used for functionality that is removed but only for that that is still supported but discouraged. With `additionalProperties:false` we could express both removed and deprecated properties.

Reply:
I can confirm this works and implement by https://github.com/vuejs/language-tools/commit/777b0a16706d30d716fa04d24e82e6d298a3fa7f, thanks!